benchmarks/kubectl-mtb/README.md

```diff
@@ -42,77 +44,81 @@ $ kubectl-mtb get benchmarks
 ### Run the available benchmarks:
 
 ```bash
-$ kubectl-mtb run benchmarks -n "tenantnamespace" --as "user impersonation"
+$ kubectl-mtb run benchmarks -n "namespace" --as "user impersonation"
 ```
-You can mention the profile level of the  benchmark using `-p` flag. 
+You can mention the profile level of the  benchmark using `-p` flag. Users can switch to development mode by passing **debug** arg to `-v` flag. 
```

Comment:
see comments in other issue....would prefer if we use "--debug" if there is a single debug level, or something like "-v 3" if we have different levels.